recipe/meta.yaml

```
@@ -12,6 +12,7 @@ source:
 
 build:
   number: 1
+  skip: True  # [py>=3.10]
```

```suggestion
  skip: True  # [py==310]
```

I think `[py>=3.10]` skips all python builds. This change should indicate that only Python 3.10 is skipped, assuming that is what we want.

---

`py` is always an `int`: [see here](https://github.com/conda/conda-build/blob/master/conda_build/metadata.py#L98).

---

thanks... there are dependency issue with Python 3.10 right now.. we only need conda-build updated at the moment for Python 3.9 on the build machines.... feel free to remove the skip, if you want to resolve the Python 3.10 dependency issues.